utils/manifest-utils/src/util.rs

```diff
@@ -260,16 +292,15 @@ impl KeystoreManager {
         cert_path: &PathBuf,
         certs: Vec<X509>,
     ) -> anyhow::Result<()> {
-        let file_stem = get_file_stem(&cert_path)
-            .ok_or_else(|| "Cannot get file name stem")
-            .map_err(|err| anyhow::anyhow!(err))?;
-        let dot_extension = get_file_extension(&cert_path)
-            .map(|ex| format!(".{}", ex))
+        let file_stem = get_file_stem(cert_path)
+            .ok_or_else(|| anyhow::anyhow!("Cannot get file name stem."))?;
+        let dot_extension = get_file_extension(cert_path)
+            .map(|ex| format!(".{ex}"))
             .unwrap_or(String::from(""));
         for cert in certs.into_iter() {
             let id = cert_to_id(&cert)?;
             let mut new_cert_path = self.cert_dir.clone();
-            new_cert_path.push(format!("{}.{}{}", file_stem, id, dot_extension));
+            new_cert_path.push(format!("{file_stem}.{id}{dot_extension}"));
             let cert = cert.to_pem()?;
             fs::write(new_cert_path, cert)?;
         }
```

Comment:
[not this PR] I know that you didn't change / add test in this PR here but I think you are missing:
```rust
#[cfg(test)]
mod tests {
```

Reply:
Added module.